fahrschein-http-api/src/main/java/org/zalando/fahrschein/http/api/RequestHandler.java

```diff
@@ -0,0 +1,10 @@
+package org.zalando.fahrschein.http.api;
+
+public interface RequestHandler {
+
+    void beforeExecute(Request request);
+
+    void afterExecute(Request request);
```

Comment:
```suggestion
    void afterExecution(Request request);
```

Reply:
Maybe the method should also take the `response` object. ?